src/VCAuthn/Migrations/StorageDb/StorageDbContextModelSnapshot.cs

```diff
@@ -0,0 +1,64 @@
+﻿// <auto-generated />
+using System;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Storage.ValueConversion;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Metadata;
+using VCAuthn;
+
+namespace VCAuthn.Migrations
+{
+    [DbContext(typeof(StorageDbContext))]
+    partial class StorageDbContextModelSnapshot : ModelSnapshot
+    {
+        protected override void BuildModel(ModelBuilder modelBuilder)
+        {
+#pragma warning disable 612, 618
```

Comment:
Why are we disabling warnings here?

Reply:
dotnet disables itself :) it's an auto-generated code